frontend/src/components/components.css

```diff
@@ -106,3 +106,20 @@
     margin-right: 0.8em;
     margin-bottom: 0.3em;
 }
+
+.filter-chip {
+    padding-left: 1em;
+    padding-right: 2em;
+    margin-left: 1em;
```

Comment:
This seems like a lot of room around the chip. Does it look okay?

Reply:
It looks like this:
![image](https://user-images.githubusercontent.com/30083877/88678330-582d1500-d0bc-11ea-81f2-bb8fc6c2df51.png)
Not crazy about the "info" color. Might change it to primary or light

Reply:
Those padding attributes weren't doing anything. I had them set so high because I kept increasing them to see if anything would change. Since they didn't I removed them. The margin left I kept so that the chips aren't touching each other. I also changed the color to "light" (grey bg/black text).